packages/shared-base/src/types/Routes.ts

```diff
@@ -128,6 +123,9 @@ export interface PopupRoutesParamsMap {
     [PopupRoutes.Wallet]: {
         isCreating?: boolean
         external_request?: string
+        // Unlock
+        close_after_unlock?: boolean
+        from?: string | null
```

Comment:
```suggestion
        from?: PopupRoutes | null
```

Reply:
It's not plain route path only, but also could contain query.